pkgs/tools/misc/pcp/default.nix

```
@@ -0,0 +1,22 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "pcp";
+  version = "0.4.0";
+
+  src = fetchFromGitHub {
+    owner = "dennis-tra";
+    repo = "pcp";
+    rev = "v${version}";
+    sha256 = "sha256-aZO8VuOiYhOPctFKZ6a2psJB0lKHlPc+NLy2RWDU4JI=";
+  };
+
+  vendorSha256 = "sha256-fXDOOxkl4Rws28uUmOO4MxljrJPIXgWo9YYkKazr18A=";
```

Needs to be updated now `runVend` is removed.

```suggestion
  vendorSha256 = "sha256-3bkzBQ950Phg4A9p+IjeUx7Xw7eVmUbeYnQViNjghFk=";
```

---

You cannot call `buildGoModule` without the required `vendorSha256` argument:
```
user: matthew swordfish in nixpkgs on  add-pcp [$!] via ❄️  impure (shell) took 3s 
❯ nix build .#pcp -L
warning: Git tree '/home/matthew/git/nixpkgs' is dirty
error: anonymous function at /nix/store/fsa9rjw3ni69xv10wassi686r7pgcrfz-source/pkgs/development/go-modules/generic/default.nix:3:1 called without required argument 'vendorSha256'

       at /nix/store/fsa9rjw3ni69xv10wassi686r7pgcrfz-source/pkgs/tools/misc/pcp/default.nix:3:1:

            2|
            3| buildGoModule rec {
             | ^
            4|   pname = "pcp";
```

---

My bad, I thought you were telling me to get rid of the argument on first glance.